tex/chapters/MachineLearning.tex

```diff
@@ -1,3 +1,267 @@
 %!TEX root = ../main.tex
 
 \chapter{Machine Learning}\label{cha:machinelearning}
+
+In this chapter a new approach to identifying strategies that are the same will be presented.
+A new type of tournament will be presented, and the summary of the results of this tournament will then be used to train a Machine Learning model.
+The model is then able to predict whether new strategies are the same as ones already implemented within Axelrod-Python.
+
+
+
+\section{Ashlock Tournament}\label{sec:ashlock_tourn}
+
+In an Ashlock tournament, all strategies entered play against a probe strategy for a range of parameters, and do not play each other.
+This is similar to how a numerical approximation of a fingerprint is constructed in Chapter \ref{cha:implementation}.
+The benefit of this approach is that much more data is collected than just the score of the strategy involved.
```

Comment:
This is not the benefit of the approach, the approach is exactly the same, it's just that we gather more information from the tournament.